shared/styles/css.js.flow

```diff
@@ -265,7 +265,7 @@ export type _StylesDesktop = {|
   left?: Dimension,
   letterSpacing?: string,
   lineBreak?: string,
-  lineHeight?: string,
+  lineHeight?: Dimension,
```

Comment:
we actually want this to be a string likely. people too often just pass a number to it when thats the wrong thing